src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -507,6 +507,35 @@ def set_browser_implicit_wait(self, seconds):
         implicit_wait_in_secs = robot.utils.timestr_to_secs(seconds)
         self._current_browser().implicitly_wait(implicit_wait_in_secs)
 
+    def get_browser_capabilities(self):
+        """Gets current browser capabilities from webdriver.
+
+        Returns a Dictionary with webdriver (browser) desired capabilities attributes.
+        You may also query keys providing a default value when those keys are nonexistent.
+
+        Examples of returned attributes are:
+        'rotatable','takesScreenshot', 'acceptSslCerts', 'cssSelectorsEnabled', 'javascriptEnabled',
+        'databaseEnabled', 'locationContextEnabled', 'platform', 'browserName', 'version',
+        'nativeEvents', 'applicationCacheEnabled', 'webStorageEnabled', 'handlesAlerts'
+
+        Examples:
+        | ${capabilities}= | Get Browser Capabilities                            |                           |
+        | Log              | ${capabilities.browserName} ${capabilities.version} | # Using attributes        |
+        | Log              | Capabilities = ${capabilities}                      |                           |
+        | Log              | ${capabilities['platform']}                         | # Using dictionary        |
+        | Log              | ${capabilities.get('may_not_exist', 'default')}     | # Gets 'default'          |
+        | Log              | ${capabilities['may_not_exist']}                    | # Produces KeyError       |
+        | Log              | ${capabilities.may_not_exist}                       | # Produces AttributeError |
+
+        References, see [http://selenium-python.readthedocs.org/en/latest/api.html?highlight=desired_capabilities#selenium.webdriver.remote.webdriver.WebDriver.desired_capabilities|Desired Capabilities] and [https://selenium.googlecode.com/svn/trunk/docs/api/py/webdriver/selenium.webdriver.common.desired_capabilities.html#module-selenium.webdriver.common.desired_capabilities|Common Desired Capabilities].
+
+        New in Selenium2Library 1.8.0.
+        """
+
+        capabilities = self._current_browser().capabilities
+
+        return AttrDict(capabilities)
```

Comment:
Unnecessary empty lines above. This PR removes some unnecessary empty lines elsewhere, which is fine, but shouldn't add new ones.